backend/src/main.rs

```diff
@@ -512,6 +512,15 @@ async fn user_connected(ws: WebSocket, games: Games, stats: Arc<Mutex<InMemorySt
             };
             info!(game.tracer(&logger, &room, Some(1)), "Joining room"; "player_id" => player_id.0);
             game.users.insert(ws_id, UserState { player_id, tx });
+
+            // if the same user joined before, remove its previous entry from the user list
+            if game.game.get_user_multi_game_session_policy()
```

Comment:
nit: let's do

```
if game.game.allows_multiple_sessions_per_user() {
}
```
and avoid looking at the `UserMultiGameSessionPolicy` enum at all